### DISCUSSION TITLE
[PhantomWallet][Bitcoin]: Unable to sign PSBT: contains OP_RETURN operation

### DISCUSSION THREAD

**Discussion by keygray** [phantom/docs#179] [Help & Support]
### Summary

Cannot sign psbt when psbtHex has OP_RETURN
Could anyone assist me in resolving the issue mentioned above?

### Steps to Reproduce

This is error when i call function window.phantom.bitcoin.signPSBT
<img width="358" alt="image" src="https://github.com/phantom/docs/assets/63116080/b3a539b7-b564-4ec6-9d51-d3706b2ef372">
This is error when i inspect brower of phantom wallet
<img width="1313" alt="image" src="https://github.com/phantom/docs/assets/63116080/f71ee510-2ada-4354-99a5-0b77a8710de4">

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
@keygray Would you mind providing your code around this transaction attempt, please?

**Reply by keygray** [reply]
https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/transactions.spec.ts 
I just create a transaction with an OP_RETURN output, similar to the example in the documentation. No additional requirements or special features needed
@adamdelphantom

**Reply by keygray** [reply]
SignPsbt only work if i remove addOutput of OP_Return `
  const data = Buffer.from('OP text', 'utf8')
  const embed = btc.payments.embed({ data: [data] })

  tx.addOutput({
    script: embed.output!,
    value: 0
  })`
